tools/autograd/templates/Functions.cpp

```diff
@@ -1489,7 +1489,11 @@ std::tuple<Tensor, Tensor, Tensor> prelu_double_backward(
 //
 // This makes no assumption on the signs of sigma.
 Tensor svd_backward(const std::vector<torch::autograd::Variable> &grads, const Tensor& self,
-          bool some, const Tensor& raw_u, const Tensor& sigma, const Tensor& raw_v) {
+          bool some, bool compute_uv, const Tensor& raw_u, const Tensor& sigma, const Tensor& raw_v) {
+  AT_CHECK(compute_uv,
+           "Setting compute_uv to false doesn't compute singular matrices, and hence we cannot compute backward ",
```

Comment:
Could you compute the singular matrices here by calling svd?
Alternatively, could you make it more explicit that this is coming from svd_backward? something like "svd_backward: Setting compute_uv to false in torch.svd doesn't compute singular matrices, and hence we cannot compute backward. Please use torch.svd(compute_uv=True)"

Reply:
Yes I agree with Richard!

Reply:
Sure, I will modify the error message. I will change it for `symeig` as well.